app/views/landing-pages/league/PageLeagueGlobal.vue

```diff
@@ -463,7 +463,7 @@ export default {
           </div>
           <div class="row flex-row" style="justify-content: flex-start;">
             <div class="col-sm-5">
-              <p style="margin-bottom: 70px;">
+              <p class="subheader2" style="margin-bottom: 70px;">
```

Comment:
Credit to Shasha for catching this one. It is due to this class header. Either add to the other `<p>` tags or remove from here.

![image](https://user-images.githubusercontent.com/15080861/105105447-f47f1580-5a68-11eb-9af2-0a56edaa20a4.png)

Reply:
Removed class `subheader2` from p tag